tests/e2e/longevity/test_stage1.py

```diff
@@ -0,0 +1,43 @@
+import logging
+from ocs_ci.ocs.longevity import start_ocp_workload
+from ocs_ci.framework.testlib import E2ETest, skipif_external_mode
+
+
+log = logging.getLogger(__name__)
+
+
```

Comment:
do we need ignore_leftovers here ? as we are going to keep all pvcs, pods and workloads running from stage-0 and stage-1

Reply:
No, not needed. All the resources that we create in stage-1 will be deleted at the end, so there will not be any left overs.
The stage0 resources will be present even before running stage1, so they will not be treated as leftovers in the test